hyperjumptech/monika
# feature(config): [297] fix monika json generated

# Monika Pull Request (PR)  
Fix #297 
## What feature/issue does this PR add  
The generated monika.json when converting from HAR does not include headers and body of request.

## How did you implement / how did you fix it  
add header and body parser and copy it to monika.json file generated

## How to test  
1. copy har file example from the issue #297 to `copy_example.har` file
2. run `./bin/run --create-config --har copy_example.har`
3. check generated monika.json, it should have headers and body filled.

![Screen Shot 2021-07-27 at 20 04 25](https://user-images.githubusercontent.com/8952493/127158394-bea93189-bf5d-433c-b14f-3be79d5ae1df.png)
